lib/rules/web/console/base/login.xyaml

```diff
@@ -94,10 +94,13 @@ login_token:
   action: verify_logged_in
 
 login_page_loaded:
-  element:
-    selector:
-      css: "#password, #inputPassword, .login-redhat, .idp, .navigation-container"
-    timeout: 30
+  elements:
+    - selector:
+        text: Username
+      timeout: 30
+    - selector:
+        text: Password
+      timeout: 30
```

Comment:
These changes are not needed.  `css: "#password, #inputPassword, .login-redhat, .idp, .navigation-container"` are doing the same thing